digid_eherkenning/saml2/base.py

```diff
@@ -270,23 +295,23 @@ def create_config_dict(self, conf):
                     ],
                 },
                 "NameIDFormat": "urn:oasis:names:tc:SAML:1.1:nameid-format:unspecified",
-                "x509cert": open(conf["cert_file"], "r").read(),
-                "privateKey": open(conf["key_file"], "r").read(),
+                "x509cert": certificate,
+                "privateKey": privkey,
                 "privateKeyPassphrase": conf.get("key_passphrase", None),
             },
             "idp": idp_settings,
         }
 
         telephone = conf.get("technical_contact_person_telephone")
         email = conf.get("technical_contact_person_email")
-        if telephone or email:
+        if telephone and email:
```

Comment:
@SilviaAmAm the metadata generation commands used `and`, but this had `or` initially - do you think this will cause problems?